packages/expo/virtual/streams.js

```diff
@@ -0,0 +1,8 @@
Symbol.toStringTag&&Object.defineProperty(qo.prototype,Symbol.toStringTag,{value:"TransformStream",configurable:!0});let Eo=function(){return _createClass((function TransformStreamDefaultController(){throw _classCallCheck(this,TransformStreamDefaultController),new TypeError("Illegal constructor")}),[{key:"desiredSize",get:function(){if(!Xr(this))throw eo("desiredSize");return hr(this._controlledTransformStream._readable._readableStreamController)}},{key:"enqueue",value:function enqueue(r=void 0){if(!Xr(this))throw eo("enqueue");Kr(this,r)}},{key:"error",value:function error(r=void 0){if(!Xr(this))throw eo("error");var z;z=r,Hr(this._controlledTransformStream,z)}},{key:"terminate",value:function terminate(){if(!Xr(this))throw eo("terminate");!function(r){const z=r._controlledTransformStream;dr(z._readable._readableStreamController);Vr(z,new TypeError("TransformStream terminated"))}(this)}}])}();function Xr(r){return!!t(r)&&!!Object.prototype.hasOwnProperty.call(r,"_controlledTransformStream")&&r instanceof Eo}function Jr(r){r._transformAlgorithm=void 0,r._flushAlgorithm=void 0,r._cancelAlgorithm=void 0}function Kr(r,z){const D=r._controlledTransformStream,te=D._readable._readableStreamController;if(!mr(te))throw new TypeError("Readable side is not in a state that permits enqueue");try{fr(te,z)}catch(r){throw Vr(D,r),D._readable._storedError}const re=function(r){return!ur(r)}(te);re!==D._backpressure&&Gr(D,!0)}function Zr(r,z){return _(r._transformAlgorithm(z),void 0,(z=>{throw Hr(r._controlledTransformStream,z),z}))}function eo(r){return new TypeError(`TransformStreamDefaultController.prototype.${r} can only be used on a TransformStreamDefaultController`)}function to(r){void 0!==r._finishPromise_resolve&&(r._finishPromise_resolve(),r._finishPromise_resolve=void 0,r._finishPromise_reject=void 0)}function ro(r,z){void 0!==r._finishPromise_reject&&(p(r._finishPromise),r._finishPromise_reject(z),r._finishPromise_resolve=void 0,r._finishPromise_reject=void 0)}function oo(r){return new TypeError(`TransformStream.prototype.${r} can only be used on a TransformStream`)}Object.defineProperties(Eo.prototype,{enqueue:{enumerable:!0},error:{enumerable:!0},terminate:{enumerable:!0},desiredSize:{enumerable:!0}}),o(Eo.prototype.enqueue,"enqueue"),o(Eo.prototype.error,"error"),o(Eo.prototype.terminate,"terminate"),"symbol"==typeof Symbol.toStringTag&&Object.defineProperty(Eo.prototype,Symbol.toStringTag,{value:"TransformStreamDefaultController",configurable:!0});const Wo={ReadableStream:Co,ReadableStreamDefaultController:To,ReadableByteStreamController:mo,ReadableStreamBYOBRequest:_o,ReadableStreamDefaultReader:ao,ReadableStreamBYOBReader:po,WritableStream:yo,WritableStreamDefaultController:So,WritableStreamDefaultWriter:vo,ByteLengthQueuingStrategy:Po,CountQueuingStrategy:ko,TransformStream:qo,TransformStreamDefaultController:Eo};for(const r in Wo)Object.prototype.hasOwnProperty.call(Wo,r)&&Object.defineProperty(wo,r,{value:Wo[r],writable:!0,configurable:!0})}();
```

Comment:
I think it might be nice to utilize the remaining dev dependency and write a script to generate this file. The instructions otherwise feel a little less reproducible than what's probably needed for a polyfill.

But totally happy to come back to that in the future since generating it once feels fine to merge until we need to update it or verify it

Reply:
+1 for this.